### PR TITLE
Update

### PR DESCRIPTION
**Pull request by Justman10000**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,121 +28,121 @@
...
-    "urlpattern-polyfill": "^6.0.2",
+    "url-loader": "^4.1.1",
+    "urlpattern-polyfill": "^9.0.0",
     "v-tooltip": "~2.1.3",
-    "videojs-record": "^4.5.0",
-    "vue": "2.6.12",
-    "vue-chartjs": "3.5.1",
-    "vue-clickaway": "~2.1.0",
+    "videojs-record": "^4.6.0",
+    "vue": "3.3.4",
```
Comment (LiamAshdown) [inline package.json:79]: This would completely break the web app, let alone wouldn't even compile.

Vue 3 introduces breaking changes.

Comment (Justman10000) [reply package.json:79]: Will take a closer look

**Comment by pranavrajs** [comment]
Closing this as there is no activity on this PR for sometime.

**Comment by Justman10000** [comment]
> Closing this as there is no activity on this PR for sometime.

It would be helpful to always ping before you close! But anyway, destroy quiet work of others! I think it is now more apparent why I originally left among other things

**Comment by pranavrajs** [comment]
@Justman10000 I didn't mean to offend. Keeping the PRs and issues up to date is our responsibility. This didn't add value directly to the project as the dependency updates as done in the PR doesn't work and the comment on the PR has not addressed for a while. It is natural to close it out as it feels like an abandoned PR. If it is not the case and if you feel to contribute, please feel to reopen this.

**Comment by Justman10000** [comment]
> please feel to reopen this.

Would I feel insulted if I could do this? When maintainers in repos close PRs, only they can reopen the PRs, unless, as already indicated, the PR creator himself closes the PR, then this can also open the PR again, but otherwise...

grafik

**Pull request closed without merging**